build.gradle

```diff
@@ -23,18 +23,25 @@ dependencies {
     // Fabric API. This is technically optional, but you probably want it anyway.
     modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
 
-    modApi "com.terraformersmc:modmenu:3.2.3"
-    modApi("me.shedaniel.cloth:cloth-config-fabric:6.2.62") {
+    modApi "com.terraformersmc:modmenu:5.0.2"
+    modApi("me.shedaniel.cloth:cloth-config-fabric:9.0.94") {
         exclude(group: "net.fabricmc.fabric-api")
     }
     include(implementation('club.minnced:discord-rpc-release:v3.4.0'))
     include(implementation("com.github.MinnDevelopment:java-discord-rpc:v2.0.2"))
 
-    modApi "ch.njol.minecraft:ui-framework-mc1_19:${project.ui_framework_version}"
-    include "ch.njol.minecraft:ui-framework-mc1_19:${project.ui_framework_version}"
-    modApi "ch.njol.minecraft:config-framework-mc1_19:${project.config_framework_version}"
-    include "ch.njol.minecraft:config-framework-mc1_19:${project.config_framework_version}"
+    //un-comment those if you want to load from local maven (don't have both local and jitpack sources on at the same time, dunno what it'd do I never tried it)
+    //modApi "ch.njol.minecraft:ui-framework-mc1_19_3:${project.ui_framework_version}"
+    //include "ch.njol.minecraft:ui-framework-mc1_19_3:${project.ui_framework_version}"
+    //modApi "ch.njol.minecraft:config-framework-mc1_19:${project.config_framework_version}"
+    //include "ch.njol.minecraft:config-framework-mc1_19:${project.config_framework_version}"
+    //un-comment those if you want to load from jitpack (don't have both local and jitpack sources on at the same time, dunno what it'd do I never tried it)
+    modApi('com.github.Ascynx:MCConfigFramework:mc1.19-SNAPSHOT')
+    include('com.github.Ascynx:MCConfigFramework:mc1.19-SNAPSHOT')
+    modApi('com.github.Ascynx:MCUIFramework:mc1.19.3-SNAPSHOT')
+    include('com.github.Ascynx:MCUIFramework:mc1.19.3-SNAPSHOT')
```

Comment:
Can you make PRs for these repositories as well please?
Also, which repository did you upload these artifacts to, and could I do that as well?

Reply:
it's using jitpack with my fork, would've used on the main one but it has some issues that I have fixed.

Reply:
https://jitpack.io/#Ascynx/MCUIFramework and https://jitpack.io/#Ascynx/MCConfigFramework

and I think the commits that are related to the builds are:
https://github.com/Ascynx/MCUIFramework/commit/bfc569820cecc0b8b05e02079688abe9faa5647a
and
https://github.com/Ascynx/MCConfigFramework/commit/91652ac24e4deed982097f35a02dd7968daa8682

Reply:
Thanks, will check that out!